core/src/lib/page-object-builder.ts

```diff
@@ -154,6 +166,8 @@ export class PageObjectBuilder {
                 route: scope.route,
                 hasFillForm: scope.hasFillForm
             });
+        this.options.logger.logSuccess(`✓  [appendChild]\t${instruct.name} to ${scope.name}`);
+        this.options.logger.debug('append Child to ' + parent + ' for instruct:', instruct, 'result:', child);
```

Comment:
why different ways of concatenating strings?
... + ... +....
and
${...}

Would keep it the same everywhere